org.eclipse.scanning.points/src/org/eclipse/scanning/points/MultiStepIterator.java

```diff
@@ -11,85 +11,28 @@
  *******************************************************************************/
 package org.eclipse.scanning.points;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-
 import org.eclipse.scanning.api.points.IPosition;
 import org.eclipse.scanning.api.points.ScanPointIterator;
-import org.eclipse.scanning.api.points.models.MultiStepModel;
-import org.eclipse.scanning.api.points.models.StepModel;
-import org.eclipse.scanning.jython.JythonObjectFactory;
 
 /**
  * An iterator over multiple step ranges. Acts essentially as a sequence of
  * step iterators chained together. In the special case where one step iterator begins
  * with the last value of the previous iterator it is not repeated.
  *
+ * TODO Matt D. 2017-10-26: why do we need to set exposure time? Isn't there some way
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)